src/query/context.hpp

```diff
@@ -24,6 +24,15 @@
 
 namespace memgraph::query {
 
+enum class TransactionStatus {
+  NO_TRANSACTION,
+  ALIVE,
+  CHECKING_STATUS,
+  KILLED,
+  STARTED_COMMITTING,
+  STARTED_ROLLBACK,
```

Comment:
I think we can come up here with better naming. This has to do that we use naming convention that combines adjectives and nouns to describe the state of the object:

NO_TRANSACTION -> INACTIVE
ALIVE -> ACTIVE
CHECKING_STATUS -> VERIFYING
KILLED -> TERMINATED
STARTED_COMMITING -> COMITED
STARTED_ROLLBACK -> ROLLED_BACK